api/envoy/extensions/common/ratelimit/v3/ratelimit.proto

```diff
@@ -130,3 +130,15 @@ message LocalRateLimitDescriptor {
   // Token Bucket algorithm for local ratelimiting.
   type.v3.TokenBucket token_bucket = 2 [(validate.rules).message = {required: true}];
 }
+
+// Configuration that used to enable local cluster level rate limiting where the token buckets
+// will be shared across all the Envoy instances in the local cluster.
+// A share will be calculated based on the membership of the local cluster dynamically
+// and the configuration. When the limiter refilling the token bucket, the share will be
```

Comment:
What configuration?

Reply:
The configuration from `LocalClusterRateLimit`. It's empty now because only evenly share is supported.

Reply:
@wbpcode  could you add something like your reply above `It's empty now because only evenly share is supported.`

It is not obvious for future reader. we can do it in the next PR

Reply:
Got it. Will do it with a new PR.